passkit/pubspec.yaml

```diff
@@ -8,7 +8,8 @@ topics:
   - pkpass
   
 environment:
-  sdk: ^3.0.0
+  sdk: '>=3.4.3 <4.0.0'
+  flutter: 3.22.2
```

Comment:
I don't think this change is needed. There's no need to depend on Flutter nor on a higher Dart version

Reply:
I'll remove the flutter version (got there through copy-paste) but I think we should support higher dart versions if we want to take advantage of the new language features, wdyt?

Reply:
Sounds good